plugins/woocommerce/includes/rest-api/Controllers/Version3/class-wc-rest-controller.php

```diff
@@ -596,4 +596,45 @@ function( $response_fields, $field ) use ( $fields ) {
 		);
 		return $this->_fields;
 	}
+
+	/**
+	 * Limit the contents of the meta_data property based on certain request parameters.
+	 *
+	 * Note that if both `include_meta` and `exclude_meta` are present in the request,
+	 * `include_meta` will take precedence.
+	 *
+	 * @param \WP_REST_Request $request   The request.
+	 * @param array            $meta_data All of the meta data for an object.
+	 *
+	 * @return array
+	 */
+	public function get_meta_data_for_response( $request, $meta_data ) {
```

Comment:
Yep, that's probably a good call. I think I made it public because `get_fields_for_response` is public, but I can't think of a particular reason it needs to be.

Reply:
Updated in 0e25ebee73